Makefile

```diff
@@ -1,6 +1,7 @@
 SDCC ?= sdcc
 STCCODESIZE ?= 4089
-SDCCOPTS ?= --iram-size 256 --code-size $(STCCODESIZE) --xram-size 0 --data-loc 0x30 --disable-warning 126 --disable-warning 59
+SDCCOPTS ?= --iram-size 256 --code-size $(STCCODESIZE) --xram-size 0 --data-loc 0x30 --disable-warning 126 --disable-warning 59 \
+    -DDEBUG -DWITH_ALT_LED9 -DWITHOUT_LEDTABLE_RELOC
```

Comment:
Ok here's one example of one way to parse make options. 
https://github.com/BastilleResearch/nrf-research-firmware/pull/8/files